### ISSUE TITLE
Extra <h4> tag showing up above <h3> tag that contains card title.

### ISSUE THREAD

**Issue by davemyb** [apigee/apigee-devportal-kickstart-drupal#154]
On the API cards on the home page of the demo content, there is an empty H4 tag right above the card title that is wrapped in an H3 tag. This fails WCAG 2.0 accessibility checks.

I tracked it down to src/components/card/card.twig which is doing this on line 40: `<h4 class="card-title">{{ title }}</h4>`. Then, in templates/field/field--paragraph--field-title--card.html.twig, the title is getting wrapped in an H3 or a H2 (`{% set tag = element['#object'].field_title.getValue ? 'h3' : 'h2' %}`).

So I think that the H4 wrapper should be removed from the card.twig file, leaving just `{{ title }}`. That solves the issue on my setup.

**Comment by ChristophAI** [comment]
This is also an SEO issue. In broader terms it is important for any machine-driven interpretation of the page content. The hierarchy of heading tags should always be in numerical order, so any bot can parse the page structure without applying crazy heuristics.
You will want to check the heading hierarchy on any page, and decide whether what you have makes sense overall.

**Comment by jacine** [comment]
Thanks for bringing the issue to our attention.  It's clearly a bug, and it will be resolved.
